common/translations/lang/en.json

```diff
@@ -3,7 +3,8 @@
   "data": {
     "x_CancelReplaceTx": "Cancel or Replace Transaction",
     "x_CancelTx": "Cancel Transaction",
-    "x_PasswordDesc": "This password *encrypts* your private key. This does not act as a seed to generate your keys. **You will need this password + your private key to unlock your wallet.**",
+    "x_PasswordDesc":
```

Comment:
Significant diff noise here. For now, let's use raw text instead of creating new keys in translation files (if there are any new keys)

Reply:
I'll clean up the diff noise. I think changing the keys until copy is modified would be a good thing in this case. I changed the copy for the modal 'confirm' and 'cancel' buttons, they were really long, and were causing issues even medium sized screens. 

I think it'll make a big difference in the mean time while we're in alpha / beta.